docs/cloud-tutorials/application-management.md

```diff
@@ -39,6 +39,10 @@ When you deploy an application, DBOS Cloud first runs [`npx dbos migrate`](../ap
 It then starts your application.
 After your application is deployed, DBOS Cloud hosts it at this URL, which is also printed by the deploy command:
 
+:::info
+Your application must not use more than 1GB of disk space.
```

Comment:
Rephrase it to something like: "Each app is allocated with 1GB of disk space." so it sounds more positive.